content/source/docs/enterprise/private/automated-recovery.html.md

```diff
@@ -6,35 +6,58 @@ sidebar_current: "docs-enterprise2-private-automated-recovery"
 
 # Private Terraform Enterprise Automated Recovery (Installer Beta)
 
-To provide a short Mean-Time-To-Recovery (MTTR), an installation can be configured to automatically restore the most recent snapshot.
+This guide explains how to configure automated recovery for a Private Terraform
+Enterprise installation. The goal is to provide a short Mean-Time-To-Recovery (MTTR)
+in the event of an outage. There are two steps in the automated recovery process:
 
-This is must useful when couple with using mounted disk or external services as the period of potental lost data is much shorter. If
-this mechanism is used with demo mode, all data present (database and configuration) at the last snapshot is the data restored.
+1. Configure snapshots on your current install to backup data
+1. Provision a new PTFE instance using the latest snapshot
 
-## Usage Of Scripts
+This guide will walk through both of these steps. 
 
-The expectations is that one of these scripts is run on machine boot, restoring operations of PTFE automatically.
-There are many mechanisms that can run a script on boot (cloud-init, systemd, /etc/init.d), which one is used is up
-to the user.
+## 1. Configure snapshots
 
-## Version Checking
+Snapshots are taken on the Private TFE instance. That instance can
+have two types of data on it:
+
+- Terraform Enterprise application data: The core product data such as
+run history, configuration history, state history. This data
+changes frequently.
+- Terraform Enterprise instance configuration data: The data used
+to configure Private TFE itself such as SMTP credentials, admin
```

Comment:
This isn't really true at present, SMTP creds and admin creds are not stored outside of the database.